services/projects-service/src/services/imageService.ts

```diff
@@ -93,10 +94,30 @@ export const deleteImagingProject = async (
     }
 };
 
+export const getImagingProjectReimportCount = async (
+    xnatProjectId: string,
+    dbConnection: Sequelize
+): Promise<number> => {
+    const queryResponse = await dbConnection.query<Partial<IImagingProject>>(
```

Comment:
Why use partial here? could this not just be
```suggestion
    const queryResponse = await dbConnection.query<number>(
```

Reply:
the query will return an object with the column name and the value in it, e.g. 

```
{
    "reimportCount": 2
}
```
so instead of creating a new type just for this I just used a partial of `IImagingProject` as it has the property

Reply:
yep fair enough! i didn't realise Sequelize would always return an object